packages/template-mrt-reference-app/package.json

```
@@ -1,18 +1,18 @@
 {
   "name": "template-mrt-reference-app",
-  "version": "2.8.4",
+  "version": "2.9.0-preview.0",
   "engines": {
-    "node": "^14.0.0 || ^16.0.0 || ^18.0.0",
-    "npm": "^6.14.4 || ^7.0.0 || ^8.0.0 || ^9.0.0"
+    "node": "^14.0.0 || ^16.0.0 || ^18.0.0 || ^20.0.0",
```

would this make sense? The benefit is that we don't need to do manual updates in the future?

```suggestion
    "node": ">=14",
```

---

I'll leave this for now as it clearly communicates what versions of node have been tested with v2 and there's a possibility that a future release of node will not be compatible with v2, but open to change this in a future PR